openapi/schemas/processes-job-management/inputs.yaml

```diff
@@ -0,0 +1,21 @@
+type: object
+required:
+  - inputs
+  - outputs
+properties:
+  inputs:
+    type: object
+    additionalProperties:
+      $ref: "../processes-workflow/input-workflow.yaml"
+  outputs:
+    type: object
+    additionalProperties:
+      $ref: "../processes-workflow/output-workflow.yaml"
+  links:
+    type: array
+    items:
+      $ref: "../common-core/link.yaml"
+  metadata:
+    type: array
+    items:
+      $ref: "../processes-core/metadata.yaml"
```

Comment:
Is this OAP specific?

Reply:
Mostly to provide additional provenance metadata. 
Not OAP specific.